src/Mod/Fem/Gui/TaskCreateElementSet.cpp

```diff
@@ -0,0 +1,820 @@
+/***************************************************************************
+copy of TaskCreateNodeSet.cpp
+ ***************************************************************************/
+/***************************************************************************
+ *   Copyright (c) 2013 Jürgen Riegel (FreeCAD@juergen-riegel.net)         *
+ *                                                                         *
+ *   This file is part of the FreeCAD CAx development system.              *
+ *                                                                         *
+ *   This library is free software; you can redistribute it and/or         *
+ *   modify it under the terms of the GNU Library General Public           *
+ *   License as published by the Free Software Foundation; either          *
+ *   version 2 of the License, or (at your option) any later version.      *
+ *                                                                         *
+ *   This library  is distributed in the hope that it will be useful,      *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU Library General Public License for more details.                  *
+ *                                                                         *
+ *   You should have received a copy of the GNU Library General Public     *
+ *   License along with this library; see the file COPYING.LIB. If not,    *
+ *   write to the Free Software Foundation, Inc., 59 Temple Place,         *
+ *   Suite 330, Boston, MA  02111-1307, USA                                *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "PreCompiled.h"
+#ifndef _PreComp_
+#include <QString>
+#include <QSlider>
+#include <Python.h>
+#include <memory>
+
+#include <string.h>
+#include <stdio.h>
+#include <regex>
+
+#include <Standard_math.hxx>
+
+#include <Inventor/nodes/SoEventCallback.h>
+#include <Inventor/nodes/SoCamera.h>
+#include <Inventor/events/SoMouseButtonEvent.h>
+
+#include <SMESH_Mesh.hxx>
+#include <SMESHDS_Mesh.hxx>
+#include <SMDSAbs_ElementType.hxx>
+
+#endif
+
+#include <ui_TaskCreateElementSet.h>
+#include "TaskCreateElementSet.h"
+#include <QMessageBox>
+#include <Gui/Command.h>
```

Comment:
Please check what includes are actually necessary and please keep the alphabetic sorting:
App, Base, Gui, then the .h of the .cpp file then the rest.
See the other FEM files as reference